src/core/__tests__/QueryManager/index.ts

```diff
@@ -3484,148 +3123,123 @@ describe("QueryManager", () => {
-            },
-            {
-              request: { query: query2 },
-              result: { data: data2Changed },
-            }
-          ).setOnError(reject),
-        });
+      const observable = queryManager.watchQuery<any>({ query });
+      const observable2 = queryManager.watchQuery<any>({ query: query2 });
 
-        const observable = queryManager.watchQuery<any>({ query });
-        const observable2 = queryManager.watchQuery<any>({ query: query2 });
+      const stream = new ObservableStream(observable);
+      const stream2 = new ObservableStream(observable2);
 
-        return Promise.all([
-          observableToPromise({ observable }, (result) =>
-            expect(result.data).toEqual(data)
-          ),
-          observableToPromise({ observable: observable2 }, (result) =>
-            expect(result.data).toEqual(data2)
-          ),
-        ])
-          .then(() => {
-            observable.subscribe({ next: () => null });
-            observable2.subscribe({ next: () => null });
-
-            return resetStore(queryManager).then(() => {
-              const result = getCurrentQueryResult(observable);
-              expect(result.partial).toBe(false);
-              expect(result.data).toEqual(dataChanged);
-
-              const result2 = getCurrentQueryResult(observable2);
-              expect(result2.partial).toBe(false);
-              expect(result2.data).toEqual(data2Changed);
-            });
-          })
-          .then(resolve, reject);
-      }
-    );
+      await expect(stream).toEmitMatchedValue({ data });
+      await expect(stream2).toEmitMatchedValue({ data: data2 });
 
-    itAsync(
-      "should change the store state to an empty state",
-      (resolve, reject) => {
-        const queryManager = createQueryManager({
-          link: mockSingleLink().setOnError(reject),
-        });
+      await resetStore(queryManager);
 
-        resetStore(queryManager);
+      const result = getCurrentQueryResult(observable);
+      expect(result.partial).toBe(false);
+      expect(result.data).toEqual(dataChanged);
 
-        expect(queryManager.cache.extract()).toEqual({});
-        expect(queryManager.getQueryStore()).toEqual({});
-        expect(queryManager.mutationStore).toEqual({});
+      const result2 = getCurrentQueryResult(observable2);
+      expect(result2.partial).toBe(false);
+      expect(result2.data).toEqual(data2Changed);
+    });
 
-        resolve();
-      }
-    );
+    it("should change the store state to an empty state", () => {
+      const queryManager = createQueryManager({
+        link: mockSingleLink(),
+      });
+
+      void resetStore(queryManager);
 
-    xit("should only refetch once when we store reset", () => {
+      expect(queryManager.cache.extract()).toEqual({});
+      expect(queryManager.getQueryStore()).toEqual({});
+      expect(queryManager.mutationStore).toEqual({});
+    });
+
+    xit("should only refetch once when we store reset", async () => {
```

Comment:
Could we change this to `test.skip` while we're at it so we can find skipped tests more easily in the future?
```suggestion
    it.skip("should only refetch once when we store reset", async () => {
```

Reply:
Good catch! I missed this when I went through this file.